.github/workflows/release.yml

```diff
@@ -106,18 +106,36 @@ jobs:
             echo "ASSET=$staging.tar.gz" >> $GITHUB_ENV
           fi
 
+      - name: Build RPM package
+        shell: bash
+        if: startsWith(matrix.build, 'linux-gnu')
+        run: |
+          cargo install cargo-generate-rpm
+          cargo generate-rpm --target ${{ matrix.target }}
+          mv target/${{ matrix.target }}/generate-rpm/trippy-0.7.0-dev-1.x86_64.rpm target/${{ matrix.target }}/generate-rpm/trippy-${{ needs.create-release.outputs.trip_version }}-x86_64.rpm
+      - name: Upload RPM package
+        if: startsWith(matrix.build, 'linux-gnu')
+        uses: actions/upload-release-asset@v1.0.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ needs.create-release.outputs.upload_url }}
+          asset_path: target/${{ matrix.target }}/generate-rpm/trippy-${{ needs.create-release.outputs.trip_version }}-x86_64.rpm
+          asset_name: trippy-${{ needs.create-release.outputs.trip_version }}-x86_64.rpm
+          asset_content_type: application/x-rpm
+
       - name: Create Debian package
         shell: bash
         if: startsWith(matrix.build, 'linux-gnu')
         run: |
           cargo install cargo-deb
           cargo deb --target ${{ matrix.target }} --deb-version ${{ needs.create-release.outputs.trip_version }}
           case ${{ matrix.target }} in
-            aarch64-*-linux-*) DPKG_ARCH=arm64 ;;
```

Comment:
@OmarAtefThabet can you fix the indentation here?

Reply:
I did fix the indentation here

Reply:
I meant that these 5 lines are still showing as a diff in this pr, it seems you have removed the indentation? If you can put it back so that these lines are not modified by this pr that would be ideal.